docs/user-guide/sync-options.md

```diff
@@ -142,7 +142,11 @@ spec:
 
 If the `Replace=true` sync option is set the Argo CD will use `kubectl replace` or `kubectl create` command to apply changes.
 
-This can also be configured at individual resource level.
+!!! warning
+      During the sync process, the resources will be synchronized using the 'kubectl replace/create' command.
+      This sync option has the potential to be destructive and might lead to resources having to be recreated, which could cause an outage for your application.
+      This can also be configured at individual resource level.
```

Comment:
This line “This can also be configured at individual resource level.” can be on the next line, outside the warning bog. Then we are good.

Reply:
cc @crenshaw-dev

Reply:
Done